README.md

```diff
@@ -1,6 +1,8 @@
 Pattern Matching
 ================
 
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/gvanrossum/patma/master?urlpath=lab)
```

Comment:
```suggestion
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/gvanrossum/patma/master?urlpath=lab/tree/playground-622.ipynb)
```

Reply:
so that the notebook will be open by default

Reply:
That should have been unnecessary as it is _lab_, and we have a workspace file. /tree is for classic notebook. It's just ignore AFAICT.